src/os_sensor_nodelet.cpp

```diff
@@ -409,6 +414,8 @@ class OusterSensor : public OusterClientBase {
     void create_publishers(ros::NodeHandle& nh) {
         lidar_packet_pub = nh.advertise<PacketMsg>("lidar_packets", 1280);
         imu_packet_pub = nh.advertise<PacketMsg>("imu_packets", 100);
+        config_pub = nh.advertise<std_msgs::String>("config", 1, true);
```

Comment:
additionally, I think it would be better if we create this publisher very early on in the node.. probably right after we create the client and before we invoke the `update_config_and_metadata`. This publisher serves a different purpose than the two other publishers. This configuration the two other publishers are sending data.